### PR TITLE
PR for #1: updated clean_data.py

### LINKED ISSUE #1: displaying percentage in histogram (instead of counts)
This is a practice task. The goal is to modify the ./data/code/clean_data.py in the data folder so that the histogram in output/chips_sold.pdf displays percentages instead of counts.

### PR DESCRIPTION
**Pull request by bettyyw0120**
Following [Practice Task](https://github.com/gslab-econ/lab-manual/wiki/Practice-Task), I edited the plot_data function in [~data/code/clean_data.py](https://github.com/bettyyw0120/template/blob/1-displaying-percentage-in-histogram-instead-of-counts/data/code/clean_data.py) so that the output histogram in [~data/output/chips_sold.pdf](https://github.com/bettyyw0120/template/blob/1-displaying-percentage-in-histogram-instead-of-counts/data/output/chips_sold.pdf) displays percentages instead of counts. 

Reviewers: @snairdesai @jc-cisneros @ShiqiYang2022

### CODE REVIEW

File: data/code/clean_data.py [hunk 1]
```diff
@@ -9,9 +11,21 @@ def main():
     df = clean_data(df)
     df.to_csv('output/data_cleaned.csv', index = False)
 
+"""
 def plot_data(df):
     plt.hist(df['chips_sold'])
     plt.savefig('output/chips_sold.pdf')
+"""
+def to_percent(y, position):
```
Comment (jc-cisneros) [inline data/code/clean_data.py]: Minor: Is the second argument used?

**Review by jc-cisneros — CHANGES_REQUESTED** [review]
Thanks for the work @bettyyw0120! This looks great. Can you incorporate the updated figure in the paper in the `paper_slides` module? I left a minor comment in the code.

File: data/code/clean_data.py (hunk 1 above)
Comment (bettyyw0120) [reply data/code/clean_data.py]: updated!

**Review by jc-cisneros — APPROVED** [review]
Thanks @bettyyw0120! This looks good to go. The next steps are the following:
- Squash merge to master (this reduces the number of commits shown in master).
- Write a [summary post](https://github.com/gslab-econ/lab-manual/wiki/Workflow.Details#summary) on the issue.

**Comment by snairdesai** [comment]
Hi @bettyyw0120, just checking in on progress here? Do you need any support with merging and addressing the final PR comments/resolving merge conflicts in #4?

**Pull request merged**